project-docs/wave-03.md

```diff
@@ -1 +1,23 @@
-# Wave 03 - Like
\ No newline at end of file
+# Wave 03: Event Handling and Lifting Up State
+
+In this wave we will update the components to manage a **like** feature. 
```

Comment:
I actually had a challenging time deciding how to set up the like button. I looked at the hints, fumbled a few times.

I wonder if it was challenging for me because I am rusty on our Learn curriculum. But I came up with a different solution, so I am going to push it up just in case students might have this similar idea

Reply:
just to be clear, you're going to push up a second solution branch? sounds good to me!